src/main/java/duke/core/Ui.java

```diff
@@ -240,10 +241,15 @@ public int chooseTaskToDelete(int numberOfTasks) {
      * @param patient it contains patient's info
      * @return true if user confirmed the deletion. False otherwise.
      */
-    public boolean confirmPatientToBeDeleted(Patient patient) {
-        showPatientInfo(patient);
+    public boolean confirmPatientToBeDeleted(Patient patient, boolean withTasksAssigned) {
+        //showPatientInfo(patient);
         while (true) {
-            System.out.println("The patient is to be deleted. Are you sure (Y/N)? ");
+            if (withTasksAssigned) {
+                System.out.println("All the tasks assigned to this patient will be "
```

Comment:
Since we haven't implement that multiple patients are assigned to a same task function, Kejun can't really do anything about it. I will try to add in this feature by next milestone